ktor-server/ktor-server-host-common/jvm/src/io/ktor/server/engine/BaseApplicationEngine.kt

```diff
@@ -30,21 +30,36 @@ public abstract class BaseApplicationEngine(
      */
     public open class Configuration : ApplicationEngine.Configuration()
 
+    private var isFirstLoading = true
+
     init {
+        var initializedStartAt = System.currentTimeMillis()
         BaseApplicationResponse.setupSendPipeline(pipeline.sendPipeline)
         environment.monitor.subscribe(ApplicationStarting) {
+            if (!isFirstLoading) {
+                initializedStartAt = System.currentTimeMillis()
+            }
             it.receivePipeline.merge(pipeline.receivePipeline)
             it.sendPipeline.merge(pipeline.sendPipeline)
             it.receivePipeline.installDefaultTransformations()
             it.sendPipeline.installDefaultTransformations()
             it.installDefaultInterceptors()
         }
         environment.monitor.subscribe(ApplicationStarted) {
+            val finishedAt = System.currentTimeMillis()
             environment.connectors.forEach {
                 environment.log.info(
                     "Responding at ${it.type.name.lowercase(Locale.getDefault())}://${it.host}:${it.port}"
                 )
             }
+
+            val elapsedTimeInSeconds = (finishedAt - initializedStartAt) / 1_000.0
+            if (isFirstLoading) {
+                environment.log.info("Started application in $elapsedTimeInSeconds seconds.")
```

Comment:
I believe `Application started in` sounds more natural than `Started application in`